packages/react/src/components/FlyoutMenu/FlyoutMenu.test.e2e.jsx

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import { mount } from '@cypress/react';
+import { onlyOn } from '@cypress/skip-test';
+
+import FlyoutMenu, { FlyoutMenuDirection } from './FlyoutMenu';
+
+const AllFlyoutMenusForPositioning = (isRTL) => (
+  <div
+    data-testid="flyout-positioning-test"
+    style={{
+      display: 'flex',
+      flex: '1',
+      flexDirection: 'column',
+      height: '100vh',
+      padding: '4rem',
+    }}
+  >
+    <div style={{ display: 'flex', flex: 1, flexDirection: 'row' }}>
+      <div
+        style={{
+          display: 'flex',
+          alignItems: 'flex-start',
+          justifyContent: 'flex-start',
+          flex: 1,
+        }}
+      >
+        <FlyoutMenu
+          direction={FlyoutMenuDirection.LeftEnd}
+          useAutoPositioning
+          menuOffset={{ top: 2, left: isRTL ? -2 : 0 }}
```

Comment:
I've had to manually adjust the positioning of these with menuOffset objects. They are off by a few pixel in Cypress, but look perfect in storybook. I don't know if these offsets will need to be adjusted over time, or if I should leave them off and take the snapshots with them in the "wrong" position since they are correct in storybook/chrome.

Reply:
I also tested in the Storybook iframe canvas where it looked good, it is strange that they are off here. If you leave the adjustments here we should add a comment like the above to the code that explains why.

Reply:
Yeah, I spent quite-a-while yesterday trying different padding on the cypress window, different sizes, etc. I don't know. I'll copy my comment into the code, though.

Reply:
That is aggravating!

Reply:
I wonder if these offsets would be necessary if we used grid instead of flexbox (just an example). Wondering if its just a weird bug from Cypress's implementation